README.md

```diff
@@ -1,28 +1,109 @@
 # Market Maker Tools
 
 ## Arby
-A market making bot that allows its users to easily arbitrage between OpenDEX and centralized exchanges.
+Arby is a market maker bot that allows its users to easily arbitrage between [OpenDEX](http://opendex.network/) and centralized exchanges. Users with spare capital are able to "earn interest" on their assets by providing liquidity to the OpenDEX network.
 
-Arby looks for arbitrage opportunities on the OpenDEX network. It will enter active orders with the configured margin to the orderbook and updates them as soon as the price changes.
+The purpose of Arby is to pull liquidity from centralized exchanges into OpenDEX while incentivizing liquidity providers (market makers) financially.
+
+### How it works
+Arby looks for arbitrage opportunities on the OpenDEX network. It will issue orders with the configured margin to the orderbook and update them as soon as the price changes.
```

Comment:
```suggestion
Arby looks for arbitrage opportunities on the OpenDEX network. It issues orders on OpenDEX based on the price of a connected centralized exchange, adding the configured `margin` on top. It will update the orders as soon as the price on the centralized exchange changes.
```